src/modules/flow/netctl/netctl.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "netctl",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "Apache-2.0",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "netctl",
+      "description": "Send list of all network services attached to board.",
+      "methods": {
+        "open": "open_list_services",
+	"close": "close_list_services"
+      },
+      "name": "netctl/list-services",
+      "out_ports": [
+        {
+          "data_type": "string",
+          "description": "Name of network services supported by system.",
+          "name": "OUT"
```

Comment:
change this to `ADDED` port, then add a `REMOVED` port. This way you keep the list-services dynamic. Also document this on its `description`, mentioning that on open, existing services will be dispatched at `ADDED`, then when new services appear, they will be dispatched on that same port, while services that are gone (ie: access points, technologies that were turned off) go in `REMOVED` port.